.circleci/config.yml

```diff
@@ -25,11 +25,8 @@ jobs:
           paths:
             - Carthage
       - run:
-          name: Run Tests (Swift 4)
```

Comment:
The tests should still run on the latest Swift 4.2

Reply:
Agreed.